melodic/distribution.yaml

```diff
@@ -9224,6 +9224,21 @@ repositories:
       type: git
       url: https://github.com/ethz-asl/rotors_simulator.git
       version: feature/gazebo9-autobackport
+  rowma_ros:
+    doc:
+      type: git
+      url: https://github.com/rowma/rowma_ros.git
+      version: master
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/rowma/rowma_ros.git
+      version: 0.0.2-1
+    source:
+      type: git
+      url: https://github.com/rowma/rowma_ros.git
```

Comment:
Same question as https://github.com/ros/rosdistro/pull/24451#discussion_r407467716

Reply:
I fixed the license at package.xml to GPLV3.